src/utils/findGitmojiCommand.js

```diff
@@ -19,14 +29,23 @@ const getOptionsForCommand = (command: ?string, flags: Object): ?Object => {
 
 const findGitmojiCommand = (cli: any, options: Object): void => {
   const flags = cli.flags
+
   const commandFlag = Object.keys(flags)
     .map((flag) => flags[flag] && flag)
     .find((flag) => options[flag])
-  const commandOptions = getOptionsForCommand(commandFlag, flags)
 
+  const commandOptions = getOptionsForCommand(commandFlag, flags)
   return options[commandFlag]
     ? options[commandFlag](commandOptions)
     : cli.showHelp()
 }
 
+const omit = (obj: Object, ...props: mixed[]) => {
```

Comment:
Hey! Not sure if I follow, can we use here?

https://github.com/sindresorhus/meow#allowunknownflags

To forward all of the unknown flags to the `git` command?

Reply:
@carloscuesta have you found some time to do some further investigations? Am still sure that https://github.com/sindresorhus/meow#allowunknownflags does not help with the current code because currently the unknown flags are already forwarded but the business logic within the code has a mapping part and does (not yet) map unknown flags